### ISSUE TITLE
THunder Client failed to respond

### ISSUE THREAD

**Issue by debasistripathy01** [rangav/thunder-client-support#992]
**Your Team Size Using TC:**

*
**Question:**
Command 'Thunder Client: Change Environment' resulted in an error (command 'thunder-client.change-env' not found)

Why am I getting this error repeatedly ?

**Comment by rangav** [comment]
Thanks @debasistripathy01 for reporting the issue, will look into.

Also if you can share steps to reproduce the error also will help to identify the source of problem

**Comment by rangav** [comment]
What version of vscode and extension are you using?

**Comment by debasistripathy01** [comment]
- [ ] uninstalled and re-installed the whole Extension 
- [ ] using command pellet of VS-code tried to open  thunder client. I was getting this error
- [ ] If any extension need to be manually deleted from the .VS-code file then please let me know

**Comment by rangav** [comment]
What version of extension are you using?

**Comment by debasistripathy01** [comment]
Version: 1.74.3 (Universal)

Node.js: 16.14.2
V8: 10.2.154.15-electron.0
OS: Darwin x64 21.6.0

**Comment by rangav** [comment]
Extension version?

**Comment by debasistripathy01** [comment]
v2.3.4

**Comment by rangav** [comment]
Thanks will look into it.

**Comment by debasistripathy01** [comment]
Thank you for looking into it

**Comment by rangav** [comment]
I tested few commands from command palette, all working ok to me.

Can you let me know which `command` are you running in command paletter?

**Comment by debasistripathy01** [comment]
Yes I am adding some screenshots of the Issue. Also I fixed the issue. 
At first everything was perfect but when I updated my VS code. Two extensions stopped working. One was Live Server and other one is thunder Client. I looked into the  .VS code > extensions  folder. There were some files already stored even after uninstalling the extensions. For the live server I reinstalled after deleting a file named "ritwickdey.liveserver-5.7.9" and same for the thunder client with file named "rangav.vscode-thunder-client-2.3.4". I re-installed TC. It's working now. 
![Screenshot 2023-01-31 at 4 09 00 AM](https://user-images.githubusercontent.com/104076012/215612608-493d6957-7440-471c-b00b-88f13fd27e9d.png)
![Screenshot 2023-01-31 at 4 09 30 AM](https://user-images.githubusercontent.com/104076012/215614704-8c61a532-3e99-4ae2-a815-6a691fcd2d79.png)
